stp_zmq/zstack.py

```diff
@@ -675,6 +677,11 @@ def send(self, msg: Any, remoteName: str = None, ha=None):
                 # Serializing beforehand since to avoid serializing for each
                 # remote
                 msg = self.serializeMsg(msg)
+                try:
+                    self.checkMsgLenWithException(msg)
```

Comment:
Why don't we do the check at the very beginning? No need to re-check it in `transmit` and `transmitThroughListener`.